manifests/candlepin.pp

```diff
@@ -50,7 +53,7 @@
     keystore_password            => $certs::candlepin::keystore_password,
     truststore_file              => $certs::candlepin::truststore,
     truststore_password          => $certs::candlepin::truststore_password,
-    artemis_client_dn            => $certs::candlepin::artemis_client_dn,
```

Comment:
Does the prior existence of this not already imply the behavior you mentioned trying to avoid with this change? This relies on `certs::candlepin` which in turn relies on `certs::foreman` -- https://github.com/theforeman/puppet-certs/blob/master/manifests/candlepin.pp#L27

Reply:
Yes, I do think that eventually `certs::candlepin` should no longer include `certs::foreman`. It'll be a breaking change, but worth it.